src/groovy/org/pih/warehouse/tablero/ActionMenuItem.groovy

```diff
@@ -0,0 +1,22 @@
+package org.pih.warehouse.tablero
+
+class ActionMenuItem implements Serializable {
```

Comment:
This could be called simply MenuItem to be more versatile, but I am not sure if we need a class for this. I guess this could be a simple json object inside config. (Unless it was specifically requested to have its own class)